m4_sessions/1804inning/public_data.md

```diff
@@ -5,12 +5,56 @@ Anna Royer
 Sabrina Kennedy
 Jared Eklin
 
-Name of Session: 
+#### Name of Session: Using Public Data for Civic Tech
 
-Session Description (Why should students come to this session):
+#### Session Description (Why should students come to this session):
+Discussing and Engaging with material and others on what civic tech and public data are and how to use them to solve problems irl.
 
-Session Outcomes (What will students learn/leave with?):
+#### Session Outcomes (What will students learn/leave with?):
+* Students will be emboldened to become involved in Civic Tech projects
+* Students will know what Civic Tech is and how it is being used or has been used in the past
+* Students will have concrete resources to becoming involved in Civic Tech
+* Students will brainstorm ideas of ways to become involved in Civic Tech
+ 
+#### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Ice Breakers** (7 minutes)
+TBD
 
-Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+**Intro** (10 minutes)
+* What is Civic Tech?
+* Real examples of how people have used and are using public data for civic tech
+* What classifies data as "public"
+* What is an API? 
+
+**Open Questions**(5 minutes)
+ 
+**Breakout** (7 minutes)
+(4 small groups: each group discusses one of the following questions)
+* What kind of problems could you solve with civic tech?
+* Where can you find public data?
+* What barriers are there to access for the public data?
+* Who is in the space of civic tech?
+
+**Whole Group Share**(10 minutes)
+
+**Breakout**(10 minutes)
+(4 small groups)
+ * Think of a problem that could be solved with public data
+ * How you might attack the problem?
+
+**Whole Group Share**(10 minutes)
+  
+**Breakout**(10 minutes)
+(4 small groups: each group is given a civic tech project to research)
+ * Civic tech projects (TBD)
+ * What problem were they trying to solve?
+ * Who does it help and how?
+ * What data sources did they use?
+ * Is their solution effective and / or succesful?
+ * Who are the contributers -> how has the work been shared?
+ 
+**Whole Group Share**(10 minutes)
+
+**Wrap It Up**
```

Comment:
Be more specific here...how will you wrap it up?  Making commitments?  Telling people to join Palhka? What action steps?